### PR TITLE
- added typeaheadOnBlur event

### PR DESCRIPTION
**Pull request by imribarr-compit**
- added documentation for typeaheadOnBlur
- removed tab event key since blur event handles it
- added getter to _active item

### CODE REVIEW

**Comment by SergeyKuryatnick** [comment]
closes #1636

File: src/typeahead/typeahead.directive.ts [hunk 1]
```diff
@@ -159,13 +162,6 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
       e.preventDefault();
       return;
     }
-
-    // if tab default browser behavior will select next input field, and
-    // therefore we should close the items list
-    if (e.keyCode === 9) {
```
Comment (valorkin) [inline src/typeahead/typeahead.directive.ts:165]: all looks good except of this change, it should not be a part of adding onBlur output

**Review by valorkin — CHANGES_REQUESTED** [review]

**Comment by valorkin** [comment]
@imribarr-compit looks good to me, but small change needed

**Comment by valorkin** [comment]
BTW if you can add small test for this, it would be just awesome

File: src/typeahead/typeahead.directive.ts (hunk 1 above)
Comment (imribarr-compit) [reply src/typeahead/typeahead.directive.ts:165]: I removed it since blur event handles it already, with this line, my change doesn't work.

Comment (imribarr-compit) [reply src/typeahead/typeahead.directive.ts:165]: Also, the only thing this line does is hides the results. This happens when the user blurs the component as well.

Comment (imribarr-compit) [reply src/typeahead/typeahead.directive.ts:165]: @valorkin

**Comment by imribarr-compit** [comment]
@valorkin I've added a test for this. If you could just reply on my change request that would be great.

Thanks.

**Review by valorkin — APPROVED** [review]

**Comment by valorkin** [comment]
Looks good to me!

**Pull request merged**